### ISSUE TITLE
Manual queue of VAMC editor notifications email

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12870]
## Background

The first time the notifications email will be sent out to VAMC editors, this will be done manually. The targeted date for this manual push is Monday 4/10.

## Acceptance Criteria

- [x] Manually send notifications email
- [x] `drush php-eval "print_r(\Drupal::service('va_gov_notifications.outdated_content')->checkForOutdatedVamcContent());"`

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by laflannery** [comment]
What are we doing when/if we get kick backs?
- error reporting for queue
Create separate issue for what we do with all of this
- technical failures
- bounces cause no longer here

**Comment by laflannery** [comment]
We need to hold on this:
- Jessica Schaffer asked Facilities to provide analytics but this ask was lost in the shuffle a bit so we are blocked on this until we can provide these to her

**Comment by dawnpruitt** [comment]
Didn't mean to move this one back to stretch - wrong ticket number

**Comment by laflannery** [comment]
There was an issue with the relative links that was discovered once the emails went out. This should be resolved when the next email is sent next month. [Slack thread here](https://dsva.slack.com/archives/CT4GZBM8F/p1681217460151819)
